lib/windows/launcher/actions/desktopShortcutActions.js

```diff
@@ -90,13 +90,20 @@ function createShortcutForWindows(app) {
     return dispatch => {
         const fileName = getFileName(app);
         const filePath = path.join(config.getDesktopDir(), `${fileName}.lnk`);
-        if (app.iconPath) {
-            // Convert icns file for the app
-            const icoPath = path.join(app.path, 'icon.ico');
-            const input = fs.readFileSync(app.iconPath);
-            const output = png2icons.PNG2ICO_PNG(input, png2icons.BILINEAR, false, 0);
-            if (output) {
-                fs.writeFileSync(icoPath, output);
+        if (app.shortcutIconPath) {
+            // TODO: This implementation is duplicated due to compatitbility, remove it later.
+            const suffix = app.shortcutIconPath.substring(3);
+            let icoPath;
+            if (suffix === 'ico') {
+                icoPath = app.shortcutIconPath;
+            } else {
+                // Convert icns file for the app
```

Comment:
Comment is wrong here. Converting png to ico in this case.

Reply:
Fixed